scripts/hid_configurator/gui/gui.py

```diff
@@ -123,14 +142,25 @@ def show_load_list(self):
     def load_list(self, files_path):
         print(files_path)
         self.filepath = files_path[0]
-        self.root.ids.dfu_buttons.ids.choose_file_button.disabled = True
         self.root.ids.dfu_buttons.ids.start_uploading_button.disabled = False
+
+        if '\\' in self.filepath:
```

Comment:
Is it needed? Why don't you use os.path?

Reply:
Thanks for suggestion. Changed.